ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/config/SinkConfig.java

```diff
@@ -362,6 +365,8 @@ public static Output getOutput() {
             .build()),
         getInputCompression(output.env));
     output.env.requireAllVarsUsed();
+    // Setup OpenCensus stackdriver exporter after all measurement views have been registered.
+    StackdriverStatsExporter.createAndRegister();
```

Comment:
Can you add a link here to relevant OpenCensus docs that talk about restrictions on when this should be called?